client/views/msg_condensed.tpl

```diff
@@ -0,0 +1,7 @@
+<div class="msg {{type}} closed" data-time="{{time}}">
+	<span class="time hide-text">{{tz time}}</span>
+	<span class="from"><span class="expander"></span></span>
```

Comment:
Relying on blank space to be clickable is a user experience issue, first because there is no discoverability or affordance/feedback (e.g. no hover effect, visual indication, etc.), and also because it's easy to click by accident: users constantly click on blank space as a way to exit the current interaction, such as closing a modal, removing input focus, closing a side navigation menu, etc.

So I tried something, this is how it looks:

<img width="106" alt="screen shot 2017-08-06 at 12 32 14" src="https://user-images.githubusercontent.com/113730/29005288-aa212a10-7aa5-11e7-8dcb-4390b6936469.png">

<img width="178" alt="screen shot 2017-08-06 at 12 32 21" src="https://user-images.githubusercontent.com/113730/29005289-aa2c3a4a-7aa5-11e7-8beb-789a9c675c9f.png">

In practice, it actually works really nicely and is very intuitive: you can expand/collapse the entries by clicking either the text or the arrow.
You won't trigger this by clicking on white space by accident (for example, to select something), which has no visible bounding box anyway.

And the day nicks are added to the text will not be an issue: the condensed message will be so much more informative than opening the details view will be something users do much more rarely (i.e. respecting principles than most recurring actions are the easiest to trigger), and when they do need to do that, they can still click on the arrow or any part of the text that is not clickable, i.e. not a colored nickname.

So to me that experiment is very conclusive, it makes much more sense as is and does not affect usability, on the contrary. It also adds a bunch of goodies, like transition on hover, and reuse the styling of the thumbnail toggle for consistency. 🎉